### ISSUE TITLE
package organization

### ISSUE THREAD

**Issue by wpfff** [kouwenhovenlab/pytopo#3]
how should we organize the packages in here, in particular tools?
do dataset-tools really deserve their own repo, or should all that stuff be grouped into pytopo? I'd prefer the latter, to avoid clutter, and requiring people to manage a ton of repos on the measurement setups.
IMO pytopo should be a general toolbox, and once things really get too independent we can move it out of there.

**Comment by astafan8** [comment]
@wpfff I agree with the proposal. For now let's keep all Delft tools here. Exception so far is only the plottr.
